docs/storage.rst

```diff
@@ -108,7 +108,13 @@ On CentOS and Red Hat systems, and other RPM based distributions the command is
 
 .. code-block:: console
 
+  sudo yum install https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/pgdg-centos96-9.6-3.noarch.rpm
```

Comment:
This command is not working on CentOS 7.9, it gives following error:
```
[centos@ip-10-0-0-60 ~]$ sudo yum install https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/pgdg-centos96-9.6-3.noarch.rpm
Loaded plugins: fastestmirror
Cannot open: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/pgdg-centos96-9.6-3.noarch.rpm. Skipping.
Error: Nothing to do

```

Reply:
URL updated, it was very old